File: algebra/sum_of_n_terms_of_gp/javascript/sum_of_GP.js
// ES5 way

function sum_of_gp(n, r, a) {
  return (a*(1-Math.pow(r,n)))/(1-r);
}

// ES6 way

let sum_of_gp = (n, r, a) => (a*(1-Math.pow(r,n)))/(1-r);

File: algebra/sum_of_n_terms_of_gp/c/sumOfGP.c
#include<stdio.h>
#include<math.h>

void main(){
  float n,r,a,sum;
  printf("Enter the first term of the GP then Common Ratio then number of terms.\n");
  scanf("%f%f%f",&a,&r,&n);
  sum=(a*(1-pow(r,n)))/(1-r);
  printf("The sum of first %.0f terms is %f\n",n,sum);
}

